.github/workflows/build.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 
       - name: Upload
         env:
-         OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN: ${{ secrets.MULTIBUILD_WHEELS_STAGING_ACCESS }}
+         OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN: ${{ secrets.ANACONDA_SCIENTIFIC_PYTHON_UPLOAD }}
```

Comment:
Since the `upload_to_anaconda_staging.sh` script looks for an environment variable, you need to sync the env name with this stanza. So either this should be
```
 ANACONDA_SCIENTIFIC_PYTHON_UPLOAD: ${{ secrets.ANACONDA_SCIENTIFIC_PYTHON_UPLOAD }}
```

or the name in `tools/upload_to_anaconda_staging.sh` should remain unchanged.

Reply:
Just to be clear: the `tools/upload_to_anaconda_staging.sh` script is used only for this windows CI job. The others use `travis-ci/build_steps.sh`.